
# message to set pid limits

uint8 DoF

float32 OutputMin
float32 OutputMax
float32 IntegralMin
float32 IntegralMax


